### ISSUE TITLE
Integration nanostores DevTools with Vue as an island in Astro fails

### ISSUE THREAD

**Issue by hipertracker** [nanostores/vue#11]
I am using the Astro framework with Vue and nanostores integrations. All is working fine, except I cannot see nanostores Devtools. They are not displayed (out of the box) under Astro DevTools or Vue Devtools. 

I guess must add nanostores DevTools as the Vue plugin with the app. use() command. The problem is that the Astro-Vue integration uses a TypeScript file on **the server side**, so nanostores DevTools can't be added there. Any hints? 

This is my repo https://github.com/hipertracker/astro-primevue-minimal

**Comment by euaaaio** [comment]
I still haven't tried Nanostoes Devtools with Astro. What error do you see in console logs?

**Comment by euaaaio** [comment]
Nanostores are written in pure JS with declaration types (.d.ts files) without bundling or building. If Nanostores works, devtools should also works.

I'm going to assume that the problem is in the versions of the installed packages. We don't yet support Vue Devtools version `next`. Try to use `@vue/devtools-api` version `6.5.1`.

**Comment by euaaaio** [comment]
Just tested in your repo. It works.

```ts
import PrimeVue from 'primevue/config';
import type {App} from 'vue';

import { devtools } from '@nanostores/vue/devtools'

import 'primeicons/primeicons.css'
import { mainStore } from '../stores/main_store';

export default (app: App) => {
    app.use(PrimeVue);
    app.use(devtools, { mainStore });
};

```

<img width="1017" alt="image" src="https://github.com/nanostores/vue/assets/5614115/51916c0c-6a28-422c-b480-8c39b856b3df">

**Comment by hipertracker** [comment]
Have you tried it under Astro integration, inside _app.ts file?

When I add `import {devtools} from '@nanostores/vue/devtools'`

it fails with 

`...node_modules/.pnpm/@nanostores+vue@0.10.0_@nanostores+logger@0.2.4_@vue+devtools-api@7.0.8_nanostores@0.9.5_vue@3.4.10/node_modules/@nanostores/vue/devtools/index.js:2
import { setupDevtoolsPlugin } from '@vue/devtools-api'
         ^^^^^^^^^^^^^^^^^^^
SyntaxError: The requested module '@vue/devtools-api' does not provide an export named 'setupDevtoolsPlugin'`

I think this is because that import is executed on the server

**Comment by hipertracker** [comment]
> I still haven't tried Nanostoes Devtools with Astro. What error do you see in console logs?

SyntaxError: The requested module '@vue/devtools-api' does not provide an export named 'setupDevtoolsPlugin'

**Comment by euaaaio** [comment]
Yeah, yeah. That's exactly what I wrote about. Try to install `@vue/devtools-api` version `6.5.1`.

**Comment by hipertracker** [comment]
> Yeah, yeah. That's exactly what I wrote about. Try to install `@vue/devtools-api` version `6.5.1`.

OK. This is fine now. There must be an issue in API v7.

**Comment by euaaaio** [comment]
I will not close the issue just yet. Will try to implement new devtools support this month.

**Comment by u4yk** [comment]
Has this been implemented yet? Looks like the new Vue devtools is live, and Nanostores support seems broken.

**Comment by euaaaio** [comment]
> Has this been implemented yet? Looks like the new Vue devtools is live, and Nanostores support seems broken.

Already working on it. Will try to release in next two days.

**Comment by somethingxdev** [comment]
> > Has this been implemented yet? Looks like the new Vue devtools is live, and Nanostores support seems broken.
> 
> 
> 
> Already working on it. Will try to release in next two days.

any updates?

**Comment by euaaaio** [comment]
Just [released v0.11.0](https://github.com/nanostores/vue/releases/tag/0.11.0) with support of Vue Devtools v7 (next).
